### DISCUSSION TITLE
How is override supposed to work for multiple deps / list-values?

### DISCUSSION THREAD

**Discussion by masklinn** [tox-dev/tox#3001] [Q&A] [answered]
I've been trying to use `--override` with `exec` in order to add otherwise unnecessary pytest plugins to debug issues in pytest (seemed convenient at the time). However I just can't get it to work right: it works fine if given a single parameter:

    tox exec -e <env> -x 'testenv.deps=pytest'  -- <command>

however every form which looks like it might have multiple parameters tries to run the thing as a command:

    $ tox exec -e <env> -x 'testenv.deps=pytest pytest-repeat'  -- <command>
    <env>: install_deps> python -I -m pip install './pytest pytest-repeat'
    ERROR: Invalid requirement: './pytest pytest-repeat'
    Hint: It looks like a path. File './pytest pytest-repeat' does not exist.

    $ tox exec -e <env> -x 'testenv.deps=pytest,pytest-repeat'  -- <command>
    <env>: install_deps> python -I -m pip install './pytest,pytest-repeat'
    ERROR: Invalid requirement: './pytest,pytest-repeat'
    Hint: It looks like a path. File './pytest,pytest-repeat' does not exist.

    $ tox exec -e <env> -x 'testenv.deps=pytest\n\tpytest-repeat'  -- <command>
    <env>: install_deps> python -I -m pip install './pytest\n\tpytest-repeat'
    ERROR: Invalid requirement: './pytest\\n\\tpytest-repeat'
    Hint: It looks like a path. File './pytest\n\tpytest-repeat' does not exist.

    $ tox e -e <env> -x 'testenv.deps=[pytest-repeat,pytest]' -- <command>
    <env>: install_deps> python -I -m pip install './[pytest-repeat,pytest]'

I have no idea how this is supposed to work, and the docs [are no help](https://tox.wiki/en/4.5.1/cli_interface.html#tox-exec---override). Repeating `-x` just overwrites the previous override.

**Comment by gaborbernat** [comment]
I don't think this is supported today.

**Comment by gaborbernat** [comment, marked as answer]
Converted to issue https://github.com/tox-dev/tox/issues/3859
